bodhi/server/push.py

```diff
@@ -73,21 +73,24 @@ def push(username, cert_prefix, yes, **kwargs):
     db_factory = transactional_session_maker()
     composes = []
     with db_factory() as session:
-        if not resume and session.query(Compose).count():
+        bodhi_composes = session.query(Compose).join(Release).filter(
+            Release.composed_by_bodhi == True
+        ).all()
```

Comment:
I understand. But lets consider situation, where user runs bodhi-push.py without "-y" and without "--resume", and there are 2 composes in failed state, one for Rawhide(composed_by_bodhi=F) and one for normal release(composed_by_bodhi=T):
- At line 81, script will ask if user want to resume 2 composes(for both releases)
- At lines 160 and 169, script will print/ask about compose/updates only for the second release(one with composed_by_bodhi=T), and silently skip the compose/updates for first release
What do you thing about this situation?

Reply:
I understand. But lets consider situation, where user runs bodhi-push.py without "-y" and without "--resume", and there are 2 composes in failed state, one for Rawhide(composed_by_bodhi=F) and one for normal release(composed_by_bodhi=T):
- At line 81, script will ask if user want to resume 2 composes(for both releases)
- At lines 160 and 169, script will print/ask about compose/updates only for the second release(one with composed_by_bodhi=T), and silently skip the compose/updates for first release
What do you thing about this situation?

Reply:
There should not ever be a ```Compose``` object for Rawhide, since Bodhi won't manage Rawhide. Composes are models that are used to track compose tasks that Bodhi is running, and Bodhi won't compose Rawhide.